core/rtmp/rtmp.go

```diff
@@ -90,7 +90,10 @@ func HandleConn(c *rtmp.Conn, nc net.Conn) {
 	log.Infoln("Incoming RTMP connected.")
 
 	pipePath := utils.GetTemporaryPipePath()
-	syscall.Mkfifo(pipePath, 0666)
+	err := syscall.Mkfifo(pipePath, 0666)
+	if err != nil && err.Error() != "file exists" {
```

Comment:
We should probably use `os.IsNotExist` here. I assume it would also be returned by `syscall.Mkfifo`.